content/en/v3/admin/setup/operator/_index.md

```diff
@@ -43,15 +43,15 @@ To create a git token for passing into the operator use this button:
 Run [jx admin operator](https://github.com/jenkins-x/jx-admin/blob/master/docs/cmd/jx-admin_operator.md) command inside the git clone of the [git repository](/v3/admin/platform/) you created previously:
 
 ```bash
-jx admin operator --username mygituser --token mygittoken
+jx admin operator --username <mygituser> --token <mygittoken>
```

Comment:
can u add a note to replace <mygituser> and <mygittoken> with the bot user name and bot token?